doc/rados/configuration/ceph-conf.rst

```diff
@@ -561,33 +561,32 @@ testing purposes, and are not recommended for use by operators.
 Runtime Changes
 ===============
 
-In most cases, Ceph allows you to make changes to the configuration of
-a daemon at runtime. This capability is quite useful for
-increasing/decreasing logging output, enabling/disabling debug
-settings, and even for runtime optimization.
+In most cases, Ceph permits changes to the configuration of a daemon at
+runtime. This can be used for increasing or decreasing the amount of logging
+output, for enabling or disabling debug settings, and for runtime optimization.
 
-Generally speaking, configuration options can be updated in the usual
-way via the ``ceph config set`` command.  For example, do enable the debug log level on a specific OSD:
+Configuration options can be updated via the ``ceph config set`` command.  For
+example, to enable the debug log level on a specific OSD, run a command of this form:
 
 .. prompt:: bash $
 
    ceph config set osd.123 debug_ms 20
 
-Note that if the same option is also customized in a local
-configuration file, the monitor setting will be ignored (it has a
-lower priority than the local config file).
+.. note:: If an option has been customized in a local configuration file, the
+   monitor setting will be ignored (it has a lower priority than the local
+   config file).
 
 Override values
 ---------------
 
-You can also temporarily set an option using the `tell` or `daemon`
-interfaces on the Ceph CLI.  These *override* values are ephemeral in
-that they only affect the running process and are discarded/lost if
-the daemon or process restarts.
+Options can be set temporarily by using the `tell` or `daemon` interfaces on
+the Ceph CLI. These *override* values are ephemeral, which means that they 
+affect only the running process and are discarded/lost if the daemon or process
```

Comment:
suggest:

which means that they affect only the current instance of the daemon and revert to persistently configured values when the daemon restarts.

Reply:
That is better.